test/functional/AccessControlCoreSpec.groovy

```diff
@@ -30,7 +30,7 @@ class AccessControlCoreSpec extends LoginTester2Spec implements ConfirmationMixi
 	}
 
 	def "user cannot access admin pages"() {
-		when: "user logins"
+		when: "me logins"
```

Comment:
Uh-oh, looks like some global text replace user -> me has touched a lot of files in this commit?